src/platforms/javascript/common/configuration/integrations/default.mdx

```diff
@@ -148,3 +148,12 @@ Sentry.init({
   },
 });
 ```
+
+
+### Dedupe
+
+_Import name: `Sentry.Integrations.Dedupe`_
+
+This integration deduplicates certain events; it can be helpful if you are receiving many duplicate errors. Be aware that Sentry will only compare stack traces and fingerprints. It is enabled by default for Browser.
```

Comment:
```suggestion
This integration deduplicates certain events. It can be helpful if you are receiving many duplicate errors. Note that Sentry will only compare stack traces and fingerprints. This integration is enabled by default for Browser.
```